front-end/src/components/Admin/Record/RecordEditor/index.js

```diff
@@ -63,6 +64,10 @@ class RecordEditor extends Component {
           loading: false
         });
         toastr.success('Данные успешно сохранены');
+
+        for (let i = 0; i < formInputs.length; i++) {
```

Comment:
Consider moving both resizeInputs from two functions to a single place in componentDidUpdate.